modules/rhcos-analyse-detect.adoc

```diff
@@ -11,3 +11,4 @@ The Compliance container was already a part of earlier {product-title-short} ver
 However, the Node-inventory container is new with {product-title-short} 4.0 and works only with {ocp} cluster nodes.
 
 Upon start-up, the Compliance and Node-inventory containers begin the first inventory scan of {op-system-first} software components within five minutes. Next, the Node-inventory container scans the node's file system to identify installed RPM packages and report on {op-system} software components. Afterward, inventory scanning occurs at periodic intervals, typically every four hours. You can customize the default interval by configuring the ROX_NODE_SCANNING_INTERVAL environment variable for the Compliance container.
+//needs updating for 4.6?
```

Comment:
No, Node Scanning (Scanner V2) and Node Indexing (Scanner V4) share these settings - both the initial delay as well as `ROX_NODE_SCANNING_INTERVAL`.
This might change in the future when we deprecate Scanner V2, though.